UnityProject/Assets/Scripts/Items/Medical/Implanter.cs

```diff
@@ -0,0 +1,137 @@
+using UnityEngine;
+using HealthV2;
+using Items;
+
+public class Implanter : MonoBehaviour, ICheckedInteractable<HandApply>, ICheckedInteractable<InventoryApply>, ICheckedInteractable<HandActivate>
+{
+	[SerializeField]
+	private GameObject implantObject = null;
+
+	[SerializeField]
+	private int timeToImplant = 5;
+
+	[SerializeField]
+	private ItemTrait ImplantableTrait = null;
+
+	[SerializeField]
+	private SpriteHandler spriteHandler;
```

Comment:
is it needed to manually set the spritehandler here? Can't we get it same we are getting ``itemStorage`` in ``Awake``?